### PR TITLE
fix: split xamarin forms events BACK into seperate package to prevent future targeting conflicts

### PR DESCRIPTION
**Pull request by ghuntley**
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**

bugfix for regression

**What is the current behavior? (You can also link to an open issue here)**

Xamarin.Forms was merged into the `reactiveui-events` NuGet package; the old behaviour was to have a seperate NuGet package called `reactiveui-events-xamforms`

**What is the new behavior (if this is a feature change)?**

With tizen and in the future other platforms targeting netstandard it's not a safe assumption to allow `Xamarin.Forms === netstandard 1.3`. This splits it back out once more as a separate NuGet package. I should have caught this as part of reviewing #1332.

**Does this PR introduce a breaking change?**

No; undoes a breaking change and provides breathing room in the future. Additionally, we haven't done a public release yet 👍 

**Please check if the PR fulfills these requirements**
- [x] The commit follows our guidelines: https://github.com/reactiveui/reactiveui#contribute
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Other information**:

### CODE REVIEW

File: src/ReactiveUI.sln [hunk 1]
```diff
@@ -9,22 +9,24 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution
 		Rebracer.xml = Rebracer.xml
 	EndProjectSection
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "ReactiveUI", "ReactiveUI\ReactiveUI.csproj", "{464CB812-F99F-401B-BE4C-E8F0515CD19D}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "ReactiveUI", "ReactiveUI\ReactiveUI.csproj", "{464CB812-F99F-401B-BE4C-E8F0515CD19D}"
```
Comment (olevett) [inline src/ReactiveUI.sln]: Little concerning these have all changed?

File: src/ReactiveUI.Events.XamForms/ReactiveUI.Events.XamForms.csproj [hunk 2]
```diff
@@ -0,0 +1,26 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>netstandard1.3</TargetFrameworks>
+    <PackageTargetFallback Condition="'$(TargetFramework)' == 'netstandard1.3'">
```
Comment (olevett) [inline src/ReactiveUI.Events.XamForms/ReactiveUI.Events.XamForms.csproj]: I think worth dropping netstandard 1.3 from the ReactiveUI.Events one to complement this.

File: src/ReactiveUI.Events.XamForms/ReactiveUI.Events.XamForms.csproj [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  </PropertyGroup>  
+
+  <ItemGroup>
+    <Compile Remove="*.cs" />
+    <None Include="*.cs" />
+    <Compile Include="../ReactiveUI.Events/SingleAwaitSubject.cs" />
+    <PackageReference Include="System.Reactive" Version="3.1.0" />
+  </ItemGroup>
+
+  <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard1.3' ">
```
Comment (olevett) [inline src/ReactiveUI.Events.XamForms/ReactiveUI.Events.XamForms.csproj]: Don't know if it's worth just dropping the conditional here? We expect this to only build for netstandard 1.3, and only for XamForms don't we?

File: src/ReactiveUI.Events.XamForms/ReactiveUI.Events.XamForms.csproj (hunk 2 above)
Comment (ghuntley) [inline src/ReactiveUI.Events.XamForms/ReactiveUI.Events.XamForms.csproj]: Agreed.

File: src/ReactiveUI.Events.XamForms/ReactiveUI.Events.XamForms.csproj (hunk 3 above)
Comment (ghuntley) [inline src/ReactiveUI.Events.XamForms/ReactiveUI.Events.XamForms.csproj]: Agreed. Let's burn this.

File: src/ReactiveUI.sln (hunk 1 above)
Comment (ghuntley) [inline src/ReactiveUI.sln]: Maybe my VS2017 is downlevel? Reverting.

Comment (olevett) [inline src/ReactiveUI.sln]: :( I don't really understand VS - and the new project (and Tests) have the same GUID.

**Pull request merged**